### PR TITLE
Learn rails

### PR DESCRIPTION
**Pull request by dtt101**
Initial contribution to the Rails section of the bootcamp.

### CODE REVIEW

File: exercises/rails/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Be able to evaluate and take over an existing project
+* Testing in Rails
+* Explore the Rails ecosystem - hosting, monitoring, deployment, gems
+* Evaluate when Rails might be an appropriate technology choice
+* Encourage further exploration
+* Write real Rails code
+
+## Overview
+
+Our '[Learning Rails](http://slides.com/dtt101/learning-rails)' presentation.
```
Comment (regularfry) [inline exercises/rails/README.md]: Worth saying "please grab one of us to walk through this, it's a bit complicated" here

Comment (dtt101) [inline exercises/rails/README.md]: Done - good call!

File: exercises/rails/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+This is intended to give you a quick overview of the Rail approach and ecosystem, and how Rail is used at Sky.
+
+It's worth giving one of the course team a shout to go through it with you and answer any questions.
+
+## Exercise 1: TV Guide
```
Comment (joshnesbitt) [inline exercises/rails/README.md]: We tend to call sub-exercises tasks to not confuse them with the top level ones.

File: exercises/rails/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The application is for the web, but intended for use on mobile phones!
+
+Take a look at the wireframe in [tv-guide.pdf](https://github.com/sky-uk/ruby-bootcamp/blob/learn-rails/exercises/rails/tv-guide.pdf). You'll only be showing what is on Now & Next.
+
+The challenge here is to give some thought to your database structure and your application URL's
+
+Here are a few acceptance criteria:
+
+ - [ ] A user can view the TV guide in chronological order
```
Comment (joshnesbitt) [inline exercises/rails/README.md]: Really like the checkbox usage here, we should have done this on previous exercises :+1:

File: exercises/rails/README.md (hunk 2 above)
Comment (dtt101) [inline exercises/rails/README.md]: Renamed - cheers!

Comment (joshnesbitt) [inline exercises/rails/README.md]: Cheers! Just noting a few things, hope you don't mind!

Comment (joshnesbitt) [inline exercises/rails/README.md]: Oh and might want to rename the folder name under this too in that case?

**Comment by joshnesbitt** [comment]
Also, don't forget to add an entry to the [tutors guide](https://github.com/sky-uk/ruby-bootcamp/blob/master/tutors.md) once you've finished.

**Comment by dtt101** [comment]
@joshnesbitt - Ok - I will merge this in and then take a new cut for the tutor update.

**Comment by joshnesbitt** [comment]
@dtt101 sounds good.

**Pull request merged**